lib/rucio/web/rest/webpy/v1/redirect.py

```diff
@@ -277,17 +268,7 @@ def GET(self, scope, name):
                             else:
                                 raise ReplicaNotFound('no redirection possible - no valid RSE for HTTP redirection found')
                         else:
-                            if select == 'geoip':
-                                rep = sort_geoip(dictreplica, client_location['ip'])
-                            elif select == 'closeness':
-                                rep = sort_closeness(dictreplica, client_location)
-                            elif select == 'dynamic':
-                                rep = sort_dynamic(dictreplica, client_location)
-                            elif select == 'ranking':
-                                rep = sort_ranking(dictreplica, client_location)
-                            else:
-                                rep = sort_random(dictreplica)
-
+                            rep = sort_replicas(dictreplica, client_location, selection=select, default='random')
```

Comment:
same here

Reply:
no, the redirect endpoint is out of scope.